api/src/services/xliff.ts

```diff
@@ -0,0 +1,277 @@
+import xliff from 'xliff';
+import { omit, groupBy } from 'lodash';
+import { InvalidPayloadException, UnprocessableEntityException } from '../exceptions';
+import { AbstractServiceOptions, Accountability, Field, Relation, SchemaOverview } from '../types';
+import { FieldsService } from './fields';
+import { ItemsService } from './items';
+
+const systemFields = ['id', 'status', 'sort', 'user_created', 'date_created', 'user_updated', 'date_updated'];
```

Comment:
We shouldn't assume these system fields exist. People can name them whatever they want

Reply:
Yeah, but it will be an alternate way to detect non-translatable fields in translations collection? I've seen in multiple places in the code that system fields were identified just through hardcoded constants...

Reply:
> I've seen in multiple places in the code that system fields were identified just through hardcoded constants...

That doesn't sound right to me. It's always using the information from the schema object, which is dynamically fetched at the beginning of the request.

Reply:
Ok, I will try then to figure out how to do it properly and will fix it

Reply:
Actually, here is what I was talking about - this is a piece of code in new-collection.vue that specifying a list of predefined system columns:
![image](https://user-images.githubusercontent.com/13006578/115940619-3e183b80-a470-11eb-8cf7-7df525601346.png)

Reply:
So I understand that in my list I should, for example, instead of 'id' use the primary key name. But regarding to other fields - I don't know how to bring them from schema... On client side, as you see, the hardcoded list used.

Reply:
Actually, I see that status and sort field probably also can be calculated... But still, what about others?

Reply:
Ah, I got it now. For Translations collection the wizard for optional system fields is not used at all.. So ok, I can assume that everything, except of key fields, is translatable, right?